lang/talon/talon-python-activator.talon

```diff
@@ -0,0 +1,7 @@
+app: vscode
+win.title: /— user/
+win.title: /— knausj_talon/
+win.title: / - user - Visual Studio Code/
+tag: user.python
+-
+tag(): user.talon_python
```

Comment:
This is a breaking change to restrict talon-related python commands to relevant projects only. This is provided as an example for discussion.